app/src/main/java/swati4star/createpdf/model/PDFOptions.java

```diff
@@ -81,5 +82,11 @@ public void setWatermark(Watermark mWatermark) {
         this.mWatermark = mWatermark;
     }
 
+    public int getmPageColor() {
```

Comment:
Remove `m` from getter and setter methods make it getPageColor() and so..

Reply:
okay